icevision/models/interpretation.py

```diff
@@ -0,0 +1,244 @@
+__all__ = [
+    "sort_losses",
+    "get_stats",
+    "compute_weighted_sum",
```

Comment:
What do you think about replacing `compute_weighted_sum`  by `get_weighted_sum`

Reply:
Sure! Will do

Reply:
Done